cwl_utils/utils.py

```diff
@@ -0,0 +1,263 @@
+import os
+import pathlib
+import subprocess  # nosec
+import sys
+import urllib.error
+import urllib.parse
+import urllib.request
+from copy import deepcopy
+from typing import (
+    Any,
+    Deque,
+    Dict,
+    List,
+    MutableMapping,
+    MutableSequence,
+    Optional,
+    Tuple,
+    Union,
+)
+
+import ruamel.yaml
```

Comment:
```suggestion
from ruamel.yaml.main import YAML
```

an adjust your other usages of `YAML` to fix the mypy complaints